### PR TITLE
[optin/puller] cosmetics, unify args, add more args for easier testing

### PR DESCRIPTION
**Pull request by gimre-xymcity**

### CODE REVIEW

File: optin/puller/puller/client/NemClient.py [hunk 1]
```diff
@@ -21,6 +21,13 @@ async def finalized_height(self):
 
 		return await self.height() - 360
 
+	async def node_network(self):
+		"""Gets node info."""
```
Comment (Jaguar0625) [inline optin/puller/puller/client/NemClient.py:25]: info => network ?

File: optin/puller/puller/client/NemClient.py [hunk 2]
```diff
@@ -21,6 +21,13 @@ async def finalized_height(self):
...
+
+		url = f'{self.endpoint}/node/info'
+		node_metadata = await self._get(url, 'metaData')
+		return Network.TESTNET if node_metadata['networkId'] == -104 else Network.MAINNET
```
Comment (Jaguar0625) [inline optin/puller/puller/client/NemClient.py:29]: yoda

File: optin/puller/tests/client/test_NemClient.py [hunk 3]
```diff
@@ -39,6 +42,7 @@ async def _process_get(self, request, response_body):
 
 	# create an app using the server
 	app = web.Application()
+	app.router.add_get('/node/info', mock_server.node_info)
 	app.router.add_get('/chain/height', mock_server.height)
```
Comment (Jaguar0625) [inline optin/puller/tests/client/test_NemClient.py:46]: swap to match other ordering

**Review by Jaguar0625 — APPROVED** [review]

**Comment by gimre-xymcity** [comment]
merged into dev

**Pull request closed without merging**